sql/dd/impl/bootstrap/bootstrapper.cc

```diff
@@ -768,6 +768,7 @@ bool DDSE_dict_init(THD *thd, dict_init_mode_t dict_init_mode, uint version) {
       // From now on all DD tables will be created with encryption='y'
     }
   }
+  delete p;
```

Comment:
A better (safer) fix would be to change the type of p to `std::unique_ptr<dd::Properties>` - grep for other uses of parse_properties for examples.